spec/services/idv/session_spec.rb

```diff
@@ -94,6 +94,7 @@
         expect(subject).to have_received(:move_pii_to_user_session)
         expect(subject.profile.active?).to eq(true)
         expect(subject.profile.deactivation_reason).to be_nil
+        expect(subject.profile.fraud_pending_reason).to be_nil
```

Comment:
Recommend adding one or several specs where `fraud_pending_reason` is not nil, combined with gpo flow and/or in person flow.